client/app/layouts/splash.html

```diff
@@ -1,3 +1,10 @@
+<style>
+md-heading-splash{
+    font-family: 'Cabin Sketch', sans-serif;
+    color: #795548;
+    font-size: 50px;
+  }</style>
+  <!-Can be added to angular-material.css>
```

Comment:
angular-material is not written by us, so we can't change it. I would recommend to put this style in the `app.styl` (supports CSS, but also the simplified [Stylus language](http://stylus-lang.com/). And maybe drop the `md-` prefix to make clear that it's our style and not from Material design :)

What about a structure like this:
```styl
.splash
  heading
    font-family: 'Cabin Sketch'
     // ... styles from splash.html
  // other styles 
```

(Further thought: maybe we should put the splash page style in a `splash.styl` next to `splash.html`... but I guess that's another task)